go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/google/go-cmp v0.5.5
 	github.com/pganalyze/pg_query_go/v5 v5.1.0
-	github.com/tetratelabs/wazero v1.6.1-0.20240124004658-4185e533bb18
+	github.com/tetratelabs/wazero v1.7.0-pre.1.0.20240311072728-64b57c75389a
```

Comment:
not sure the docker failure, but right now this branch is pointing to the unmerged branch, so I think we should pin it back to the main